src/etaoin/api.clj

```diff
@@ -1828,12 +1851,26 @@
   [driver text & more]
   (apply fill-active* driver text more))
 
-(defn fill-el
```

Comment:
@jwkoelewijn that issues with filling  elements was fixed in some prev. version. Check the latest master please.